spec.html

```diff
@@ -8968,6 +8968,8 @@ <h1>[[GetOwnProperty]] ( _P_ )</h1>
         1. If _resultDesc_.[[Configurable]] is *false*, then
           1. If _targetDesc_ is *undefined* or _targetDesc_.[[Configurable]] is *true*, then
             1. Throw a *TypeError* exception.
+          1. If _resultDesc_ has a [[Writable]] field and _resultDesc_.[[Writable]] is *false*, then
+            1. If _targetDesc_.[[Writable]] is *true*, throw a *TypeError* exception.
```

Comment:
Why do we have an "if" inside another "if"? Just combine the conditions.

Reply:
generally its just about making it easier to understand the logic for readers. the outer one contains invariants of the resultDesc and the inner one contains invariants of the targetDesc.